apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/internal/OperationCacheExtensions.kt

```diff
@@ -93,6 +94,40 @@ fun <D : Fragment.Data> Fragment<D>.readDataFromCache(
   }
 }
 
+fun <D : Operation.Data> Operation<D>.batchDataFromCache(
```

Comment:
should we be consistent here with name as rest API `.readBatchDataFromCahe`?
Also should we provide docs explaining what the purpose of this API?

Reply:
Maybe we need a parameter ? 

```
fun <D : Operation.Data> Operation<D>.readDataFromCache(
    customScalarAdapters: CustomScalarAdapters,
    readableStore: ReadableStore,
    cacheKeyResolver: CacheKeyResolver,
    cacheHeaders: CacheHeaders,
    mode: ReadMode
): D?
```

Reply:
Yeah should work 👍